src/index.html

```diff
@@ -10,6 +12,21 @@ <h2>How many do you know?</h2>
       <input type="text" id="input1" title="input1" />
       <button type="submit">Submit</button>
     </form>
+    <div>
+        <div id="progress-live" aria-live="polite" class="visually-hidden"></div>
+        <label for="progress-bar">
+            <span aria-hidden="true">Your Progress: </span>
+            <span id="progress-description" class="visually-hidden">Your progress: 0 out of 124 tags added</span>
+        </label>
+        <progress 
+            id="progress-bar" 
+            aria-valuemin="0"
+            aria-valuemax="100"
+            aria-valuetext="0"
```

Comment:
As far as I understand it only works in combination with aria-valuenow.

> The aria-valuetext attribute is used with the aria-valuenow attribute, not instead of it, unless that value is not known.
aria-valuetext is only needed when the numeric value of aria-valuenow is not meaningful.

See 👉🏽 https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuetext